ush/generate_FV3LAM_wflow.sh

```diff
@@ -840,7 +840,10 @@ if [ "${DO_DACYCLE}" = "true" ]; then
      'nggps_ic'   : false,
      'mountain'  : true,
      'warm_start' : true,
-   }"
+   }
+ 'gfs_physics_nml': {
+    'fh_dfi_radar': [${FH_DFI_RADAR[@]}],
```

Comment:
@hu5970 To accommodate changing FH_DFI_RADAR to a string,  line 845 will need to be updated as:
```
-    'fh_dfi_radar': [${FH_DFI_RADAR[@]}],
+    'fh_dfi_radar': [${FH_DFI_RADAR}],
```

Reply:
String works now and it is better solution. I have committed the changes. Thanks.